### ISSUE TITLE
Scores in log file for training

### ISSUE THREAD

**Issue by sanakhamekhem** [rwth-i6/returnn#26]
Hi,
I would like to understand the meaning of these two lines in the log file:
Is the training going ok??

Learning-rate-control: error key 'dev_score' from {'dev_error': 0.99728335186974681, 'dev_score': 3.6586107690151706}
epoch 1 score: 3.96068657248 elapsed: 3:19:03 dev: score 3.65861076902 error 0.99728335187

**Comment by albertz** [comment]
Learning-rate-control is the module which does control the learning rate (e.g. decreasing or Newbob or so), depending on some score information it gets per epoch. It just states what it stored as information for this epoch (that is `{'dev_error': 0.99728335186974681, 'dev_score': 3.6586107690151706}`). The other line just gives a summary with the same information and some more like how long this epoch took to train. Then the Learning-rate-control also states which key it uses to do its logic (in this case `dev_score`, which is probably Cross-Entropy or so). `dev_error` is most likely the frame-error-rate. So this is pretty high. But I don't know exactly what you are doing.

Note that the GitHub issue tracker is about reporting bugs. This doesn't seem like a bug, so I'm closing this now.

**Comment by sanakhamekhem** [comment]
Thank you Mr Albertz for your response
